### PR TITLE
Add support to loop a test a number of times

### PR DESCRIPTION
**Pull request by calvaris**
This functionality seems to have been lost with the last merges

### CODE REVIEW

**Comment by calvaris** [comment]
I saw this today: https://github.com/youtube/js_mse_eme/pull/13, I hope this new commit does not always enable loop.

File: js/harness/main.js [hunk 1]
```diff
@@ -39,7 +39,17 @@ var parseParams = function(testSuiteConfig) {
   config.command = parseParam('command');
   config.timeout = parseParam('timeout', TestBase.timeout);
   config.logging = !util.stringToBoolean(parseParam('disable_log', false));
-  config.loop = util.stringToBoolean(parseParam('loop', false));
+  let loop = parseParam('loop', 1);
```
Comment (tdedecko) [inline js/harness/main.js]: 'let' is not widely supported. Please don't use it.

File: js/harness/main.js [hunk 2]
```diff
@@ -39,7 +39,17 @@ var parseParams = function(testSuiteConfig) {
...
+  if (loop === 'true')
```
Comment (tdedecko) [inline js/harness/main.js]: use new method stringToBoolean

**Comment by tdedecko** [comment]
Can you add a param instead of overloading the loop param? Maybe 'loopn' or 'loopcount'.

**Comment by calvaris** [comment]
Sure I can, but how shuld any override the other? Does loop=true always spin forever? Doesn't loop=true spin forever but it is still necessary for loopcount to take effect?

**Comment by tdedecko** [comment]
loop should override loopcount since it is a switch that can be set on the UI at any time.

**Pull request closed without merging**